comment/urls.py

```diff
@@ -15,3 +16,7 @@
     path('<int:pk>/flag/state/change/', ChangeFlagState.as_view(), name='flag-change-state'),
     re_path(r'^confirm/(?P<key>[^/]+)/$', ConfirmComment.as_view(), name='confirm-comment'),
 ]
```

Comment:
```suggestion
    path('jsi18n/', JavaScriptCatalog.as_view(), name='javascript-catalog'),
]
```

and please get rid of code block below:
```python
urlpatterns += [
    path('jsi18n/', JavaScriptCatalog.as_view(), name='javascript-catalog'),
]

Reply:
since, this is different all the `URL`s that we have written, i have separated to keep it clean.

Reply:
I see it more confusing this way. I would move this path next to other paths.